GiEnJul/Controllers/AdminController.cs

```diff
@@ -132,6 +132,58 @@ public async Task<ActionResult> PutWishAsync(string person_rowkey, [FromBody] st
             return Ok();
         }
 
+        [HttpDelete("Connection")]
+        [Authorize(Policy = "DeleteConnection")]
+        public async Task<ActionResult> DeleteConnectionAsync(string location, string rowKey)
+        {
+            var giver = await _giverRepository.GetGiverAsync(location, rowKey);
+
+            if (giver?.MatchedRecipient is null)
+            {
+                return NoContent();
```

Comment:
I belive `return NotFound();` would be more fitting here

https://docs.microsoft.com/en-us/aspnet/core/web-api/action-return-types?view=aspnetcore-5.0#synchronous-action-1